orchestrator/Cargo.toml

```diff
@@ -33,13 +33,9 @@ orchestrator = { path = "./orchestrator" }
 cosmos_gravity = { path = "./cosmos_gravity" }
 ethereum_gravity = { path = "./ethereum_gravity" }
 gravity_utils = { path = "./gravity_utils" }
-gravity_proto_build = { path = "./gravity_proto_build" }
```

Comment:
The cargo setup here is structured as a workspace. These aren't unused dependencies, their local `Cargo.toml` files inform the shared overall `Cargo.lock` of the workspace. They won't build if they aren't included here.

Reply:
cargo ignores them anyway and throws warnings that they are invalid dependencies because nothing actually depends on them. they have to be built separately

Reply:
Ah, I confused this section with the `members` section.